test/fixtures/hello-cloud/bare_replica_set.yml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: extensions/v1beta1
+kind: ReplicaSet
```

Comment:
I haven't encountered a use case, but it is definitely possible to create them separately.